packages/core/src/HoverCard/HoverCardRoot.vue

```
@@ -72,27 +104,25 @@ const isPointerDownOnContentRef = ref(false)
 const isPointerInTransitRef = ref(false)
 const triggerElement = ref<HTMLElement>()
 
-function handleOpen() {
+function handleOpen(reason?: HoverCardOpenChangeReason | BaseChangeReason, event?: Event) {
   clearTimeout(closeTimerRef.value)
-  openTimerRef.value = window.setTimeout(() => open.value = true, openDelay.value)
+  openTimerRef.value = window.setTimeout(() => setState(true, reason, event), openDelay.value)
```

ESLint

[error] Pass function and arguments directly to timer function to avoid allocating an extra function

(e18e/prefer-timer-args)